orchagent/orchdaemon.cpp

```diff
@@ -313,5 +313,6 @@ void OrchDaemon::start()
         for (Orch *o : m_orchList)
             o->doTask();
 
+        flush(); //flush after each event is handled, don't wait
```

Comment:
>flush [](start = 8, length = 5)

About the performance "route performance improved by 200~300 routes/sec". Could you provide more details?
1. What is the test environment and test steps?
2. What is the performance before this PR and after?
3. If it is a small scope test, could you add a unit test or vs test case to automate it?

Reply:
OK. I just want to know performance before this PR and after? You only mentioned improved by 200~300 routes/sec. Rough numbers are ok.

Reply:
before changes it is 1300 routes/sec, platform is brcm and SAI is 3.1, CPU is Intel(R) Atom(TM) CPU  C2558  @ 2.40GHz. After this enabling pipeline changes only, it is about 1500-1600 routes/sec, if plus the syncd buffer changes in other PR, it is about 1700-1800 routes/sec